astrokat/noisediode.py

```diff
@@ -0,0 +1,67 @@
+import time
+
+from simulate import user_logger
+
+
+# switch noise-source pattern off
```

Comment:
Comment should be "on", not "off".  Also, this would be better as a docstring inside the function.